featuretools/tests/dfs_tests/test_deep_feature_synthesis.py

```diff
@@ -178,8 +178,8 @@ def test_handles_diff_entity_groupby(es):
                                    trans_primitives=[Diff])
 
     features = dfs_obj.build_features()
-    assert (feature_with_name(features, 'DIFF(value by session_id)'))
-    assert (feature_with_name(features, 'DIFF(value by product_id)'))
+    assert (feature_with_name(features, 'DIFF(value)'))
+    assert (feature_with_name(features, 'DIFF(value_many_nans)'))
```

Comment:
since these tests are checking how DFS handles using groupby with Diff we should keep testing that by using the `groupby_trans_primitives` parameter instead